### PR TITLE
Swish changes requested by Magnus Bergh

### PR DESCRIPTION
**Pull request by arebra**
Added number validation and MIT info. Did not add info about cancel, as this is not documented anywhere else either.

### CODE REVIEW

File: payments/swish/other-features.md [hunk 1]
```diff
@@ -157,6 +157,21 @@ Content-Type: application/json
...
 ```
 
+### Mobile Number Validation
+
+#### eCommerce
+
+All international mobile numbers are supported. To be valid, the number input
+must be with a country code prefix and consist of 8 to 15 characters. Digits are
+the only characters allowed, and the regex used is `\\+[1-9]\\d{7,14}`. A valid
+Swedish mobile number would be +46700123123, a valid Norwegian mobile number
```
Comment (asbjornu) [inline payments/swish/other-features.md]: Do we perhaps have a test data phone number we can use here? And if we do, should we perhaps define it as a new global variable that is used both places?

```suggestion
Swedish mobile number would be `+46700123123`, a valid Norwegian mobile number
```

File: payments/swish/after-payment.md [hunk 2]
```diff
@@ -107,7 +107,8 @@ request for the given operation.
 
 ## Swish transactions
 
-All Swish transactions are described below.
+All Swish transactions are described below. Please note that Swish does not
+support Merchant Initiated Transactions for the time being.
```
Comment (asbjornu) [inline payments/swish/after-payment.md]: Do we have somewhere MIT is described, or is that not done yet? If we do, can we link to it?

File: payments/swish/other-features.md [hunk 3]
```diff
@@ -157,6 +157,21 @@ Content-Type: application/json
...
+would be +4799999999.
```
Comment (asbjornu) [inline payments/swish/other-features.md]:
```suggestion
would be `+4799999999`.
```

File: payments/swish/after-payment.md (hunk 2 above)
Comment (arebra) [reply payments/swish/after-payment.md]: MIT is described in an include added in card payments other features. I'm on my way to add some changes on it right now, so I'll add a link now! I thought you had given it the thumbs up, but I see that only Marina did it before I merged.

File: payments/swish/other-features.md [hunk 4]
```diff
@@ -157,6 +157,21 @@ Content-Type: application/json
...
 ```
 
+### Mobile Number Validation
+
+#### eCommerce
+
+All international mobile numbers are supported. To be valid, the number input
+must be with a country code prefix and consist of 8 to 15 characters. Digits are
+the only characters allowed, and the regex used is `\\+[1-9]\\d{7,14}`. A valid
+Swedish mobile number would be `+46700123123`, a valid Norwegian mobile number
```
Comment (NullableInt) [inline payments/swish/other-features.md]:
```suggestion
Swedish mobile number would be `+46739000001`, a valid Norwegian mobile number
```

File: payments/swish/other-features.md [hunk 5]
```diff
@@ -157,6 +157,21 @@ Content-Type: application/json
...
+would be `+4799999999`.
```
Comment (NullableInt) [inline payments/swish/other-features.md]:
```suggestion
would be `+4792345678`.
```

**Pull request merged**